tools/wheels/cibw_before_build_macos.sh

```diff
@@ -21,17 +21,26 @@ if [[ $PLATFORM == "macosx-x86_64" ]]; then
   #GFORTRAN=$(type -p gfortran-9)
   #sudo ln -s $GFORTRAN /usr/local/bin/gfortran
   # same version of gfortran as the openblas-libs and scipy-wheel builds
-  curl -L https://github.com/MacPython/gfortran-install/raw/master/archives/gfortran-4.9.0-Mavericks.dmg -o gfortran.dmg
-  GFORTRAN_SHA256=$(shasum -a 256 gfortran.dmg)
-  KNOWN_SHA256="d2d5ca5ba8332d63bbe23a07201c4a0a5d7e09ee56f0298a96775f928c3c4b30  gfortran.dmg"
+  curl -L https://github.com/isuruf/gcc/releases/download/gcc-11.3.0-2/gfortran-darwin-x86_64-native.tar.gz -o gfortran.tar.gz
+
+  GFORTRAN_SHA256=$(shasum -a 256 gfortran.tar.gz)
+  KNOWN_SHA256="981367dd0ad4335613e91bbee453d60b6669f5d7e976d18c7bdb7f1966f26ae4  gfortran.tar.gz"
   if [ "$GFORTRAN_SHA256" != "$KNOWN_SHA256" ]; then
       echo sha256 mismatch
       exit 1
   fi
 
-  hdiutil attach -mountpoint /Volumes/gfortran gfortran.dmg
-  sudo installer -pkg /Volumes/gfortran/gfortran.pkg -target /
-  otool -L /usr/local/gfortran/lib/libgfortran.3.dylib
+  sudo mkdir -p /opt/
+  # places gfortran in /opt/gfortran-darwin-x86_64-native. There's then
+  # bin, lib, include, libexec underneath that.
+  sudo tar -xv -C /opt -f gfortran.tar.gz
+  sudo mkdir -p /usr/local/gfortran/lib
+  sudo cp -R /opt/gfortran-darwin-x86_64-native/lib/ /usr/local/gfortran/lib
+  ln -sf /opt/gfortran-darwin-x86_64-native/bin/gfortran /usr/local/bin/gfortran
+  export PATH=/usr/local/gfortran/lib:$PATH
+  export SDKROOT=/Applications/Xcode_11.7.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk
```

Comment:
See https://github.com/MacPython/gfortran-install/blob/47bb0df1e6f766503f81e4fd3d5d2acadd887393/gfortran_utils.sh#L93-L94

Reply:
The GHA images have quite a few SDKs installed. Wouldn't the suggestion select the most recent SDK? I was trying to use the oldest SDK so that would keep the best compatibility with older versions of macOS (old MACOSX_DEPLOYMENT_TARGET).

Reply:
Version of the SDK and the MACOSX_DEPLOYMENT_TARGET are two different things. You can use the latest SDK (13.1) and set MACOSX_DEPLOYMENT_TARGET=10.9 and it should work just fine.